registration/registration-services/src/main/java/io/mosip/registration/service/sync/impl/SyncStatusValidatorServiceImpl.java

```diff
@@ -346,13 +345,6 @@ private void captureGeoLocation(List<ErrorResponseDTO> errorResponseDTOList) {
 					"3000")));
 			geoPosition = geoPositionFacade.getMachineGeoPosition(geoPosition);
 
-			if(geoPosition.getError() != null) {
-				getErrorResponse(RegistrationConstants.ICS_CODE_FOUR,
-						geoPosition.getError(), RegistrationConstants.ERROR,
-						errorResponseDTOList);
-				return;
-			}
-
 			if (geoPosition != null && geoPosition.getError() != null) {

```

Comment:
here the condition should be (geoPosition != null && geoPosition.getError() == null)

Reply:
Changed the conditions